api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -109,6 +109,8 @@ private FhirConstants() {
 	
 	public static final String OPENMRS_FHIR_EXT_ENCOUNTER_TAG = OPENMRS_FHIR_EXT_PREFIX + "/encounter-tag";
 	
+	public static final String OPENMRS_FHIR_EXT_PRACTITIONER_TAG = OPENMRS_FHIR_EXT_PREFIX + "/practitioner-tag";
```

Comment:
I'm opening to discussion about this, as I don't really understand what this does (see below)